ocaml/xapi/xapi_vgpu.ml

```diff
@@ -119,6 +123,7 @@ let copy ~__context ~vm vgpu =
       ~other_config:all.API.vGPU_other_config
       ~_type:all.API.vGPU_type
       ~powerstate_check:false
+      ~compatibility_metadata:all.API.vGPU_compatibility_metadata
```

Comment:
So the bug here was that during a VM copy/snapshot we did not preserve the compatibility metadata?

Reply:
Yes. We missed that while developing the feature.

Reply:
..._and_ we didn't create it, which is why the patch to xapi_vm_snapshot.ml is needed.